config/opal_setup_wrappers.m4

```diff
@@ -220,18 +228,19 @@ EOF
 AC_DEFUN([OPAL_SETUP_RUNPATH],[
     OPAL_VAR_SCOPE_PUSH([LDFLAGS_save rpath_script rpath_outfile wl_fc])
 
-    AC_MSG_CHECKING([if linker supports RUNPATH])
     # Set the output in $runpath_args
     runpath_args=
     LDFLAGS_save=$LDFLAGS
     LDFLAGS="$LDFLAGS -Wl,--enable-new-dtags"
-    AC_LANG_PUSH([C])
-    AC_LINK_IFELSE([AC_LANG_PROGRAM([], [return 7;])],
-                   [WRAPPER_RPATH_SUPPORT=runpath
-                    runpath_args="-Wl,--enable-new-dtags"
-                    AC_MSG_RESULT([yes (-Wl,--enable-new-dtags)])],
-                   [AC_MSG_RESULT([no])])
-    AC_LANG_POP([C])
+    AS_IF([test x"$enable_wrapper_runpath" = x"yes"],
```

Comment:
no because by default ```$enable_wrapper_runpath``` will be set to yes.  hmm... maybe need to make it no though if enable_wrapper_rpath is also no.  I'll change that.  Forgot about the readme.  I'll fix that too.